### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by wltsmrz**

### CODE REVIEW

**Comment by scottschurr** [comment]
It's nice to have the multisign tests, and it's really nice not to have all warnings about deprecated stuff.  All tests are passing for me on OS X.

File: test/pretest.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+var fs = require('fs');
+var path = require('path');
+var joinPath = path.join.bind(path, __dirname);
+
+fs.readdirSync(joinPath('ripple-lib')).forEach(function(fileName) {
```
Comment (sublimator) [inline test/pretest.js]: I'm not super fond of this, but I imagine these tests are pretty much at the EOL at this point in time.

**Comment by sublimator** [comment]
LGTM

**Comment by ximinez** [comment]
Reviewers: @ximinez @sublimator

**Comment by ximinez** [comment]
Tests look good, and are passing in Windows and the CIs. :+1:

**Comment by nbougalis** [comment]
Merged as f2dfde3ee1a1f6fd2790a8721a17515d9368c5dc, 6066c254c8a7a68abcc04a8aca81e8272a0aef95 and 4626992474fe7e21ed31883e637c1903ae346bf5.

**Pull request closed without merging**